src/bootstrap/clean.rs

```diff
@@ -58,6 +58,9 @@ macro_rules! clean_crate_tree {
             }
 
             fn run(self, builder: &Builder<'_>) -> Self::Output {
+                if builder.build.config.dry_run() {
+                    return;
+                }
```

Comment:
isn't this already handled by `builder.run`? I'd like to still print "Cleaning ..." on `x clean --dry-run`.

Reply:
Ah true, fixed :)